versioned_docs/version-2.0/reference/cloud_architecture_and_scalability.mdx

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Cloud architecture and scalability
+
+:::note Cloud-managed solution
+This section is only relevant for the cloud-managed LangSmith service available at https://smith.langchain.com/.
+
+For information on the self-hosted LangSmith solution, please refer to the [self-hosted documentation](../self_hosting/).
+:::
+
+LangSmith is deployed on Google Cloud Platform (GCP) and is designed to be highly scalable. Many customers run production workloads on LangSmith for both LLM application observability and evaluation.
+
+## Architecture
+
+The US-based LangSmith service is deployed in the `us-central1` region of GCP.
+
+**NOTE:** We will be adding an EU instance deployed in `europe-west4` (Netherlands) soon. If you are interested in this, please contact us at sales@langchain.dev.
+
+LangSmith is composed of the following services, all deployed on Google Kubernetes Engine (GKE):
+
+- LangSmith Frontend
+- LangSmith Backend
+- LangSmith Platform Backend
+- LangSmith Playground
+- LangSmith Queue
+
+LangSmith uses the following GCP storage services:
+
+- Google Cloud Storage (GCS) for runs inputs and outputs
+- Google Cloud SQL PostgreSQL for transactional workloads
+- Google Cloud Memorystore for Redis for queuing and caching
+- Clickhouse Cloud on GCP for trace ingestion and analytics. Our services connect to Clickhouse Cloud, which is hosted in the same GCP region, via a private endpoint.
+
+Some additional GCP services we use include:
+
+- Google Cloud Load Balancer for routing traffic to the LangSmith services
+- Google Cloud CDN for caching static assets
+- Google Cloud Armor for security
+
+![](./cloud-arch.png)
+
+## Scalability
+
+LangSmith is designed to be scalable and performant.
+
+As of load testing done in February 2024, LangSmith can comfortably process 500K+ runs (spans) per minute.
+We anticipate that LangSmith can process 750K+ runs per minute with the optimizations we've made since then.
+
+![](./scalability.png)
```

Comment:
Maybe we can skip the diagram?  Would need to include more details if we wanted full diagrams / load tests

Reply:
yeah I can rm